src/package_manager/apt.ts

```diff
@@ -4,14 +4,17 @@ import * as utils from "../utils";
 
 const aptCommandLine: string[] = [
 	"DEBIAN_FRONTEND=noninteractive",
-	"RTI_NC_LICENSE_ACCEPTED=yes",
 	"apt-get",
 	"install",
 	"--no-install-recommends",
 	"--quiet",
 	"--yes",
 ];
 
+const aptCommandLineNC: string[] = [
+	"RTI_NC_LICENSE_ACCEPTED=yes",
```

Comment:
Is there any reason we shouldn't set this environment variable regardless of whether we actually choose to install the package?

Reply:
good point. even if its not installed it might still be good to explicitly mark the env var as `no`....really up to you.

Reply:
ah. that would mean we could get rid of the aptCommandLineNC all together then, yes?